contracts/modules/MigratorImmutables.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.24;
+
+import {INonfungiblePositionManager} from '@uniswap/v3-periphery/contracts/interfaces/INonfungiblePositionManager.sol';
+
+struct MigratorParameters {
```

Comment:
because itll also include v4 position manager so this struct is just preemptively there i think.
Its following the pattern of naming of 
- [UniswapParameters](https://github.com/Uniswap/universal-router/blob/228f2d151a5fc99836d72ae00f81db92cdb44bd3/contracts/modules/uniswap/UniswapImmutables.sol#L4) plus UniswapImmutables
- [PaymentParameters](https://github.com/Uniswap/universal-router/blob/228f2d151a5fc99836d72ae00f81db92cdb44bd3/contracts/modules/PaymentsImmutables.sol#L7) plus PaymentImmutables
- [RouterParameters](https://github.com/Uniswap/universal-router/blob/6cf84c8c8889661c160606a064ef03a64547f20f/contracts/base/RouterImmutables.sol#L4) plus RouterImmutables

Reply:
Could change it but at this point it would be a bigger renaming overhaul of UR 😅

Reply:
yeah i just followed the pattern from the others! and we'll have v4posm in there too

Reply:
sg